specification/elastic/Elastic.Management/ElasticMonitorResource.tsp

```diff
@@ -0,0 +1,498 @@
+  @action("listVMHost")
+  @operationId("VMHost_List")
+  vMHostList is ArmResourceActionSync<
+    ElasticMonitorResource,
+    void,
+    ArmResponse<VMHostListResponse>,
+    Error = ResourceProviderDefaultErrorResponse
+  >;
+
+  /**
+   * List detailed information about VM ingestion that will be monitored by the Elastic monitor resource, ensuring optimal observability and performance.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @action("vmIngestionDetails")
+  @operationId("VMIngestion_Details")
+  details is ArmResourceActionSync<
+    ElasticMonitorResource,
+    void,
+    ArmResponse<VMIngestionDetailsResponse>,
+    Error = ResourceProviderDefaultErrorResponse
+  >;
+
+  /**
+   * Update the VM details that will be monitored by the Elastic monitor resource, ensuring optimal observability and performance.
+   */
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @action("vmCollectionUpdate")
+  @operationId("VMCollection_Update")
+  vMCollectionUpdate is ArmResourceActionSync<
+    ElasticMonitorResource,
+    VMCollectionUpdate,
+    OkResponse,
+    Error = ResourceProviderDefaultErrorResponse,
+    OptionalRequestBody = true
+  >;
+
+  /**
+   * List all upgradable versions for your Elastic monitor resource, helping you plan and execute upgrades.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @action("listUpgradableVersions")
+  @operationId("UpgradableVersions_Details")
+  upgradableVersionsDetails is ArmResourceActionSync<
+    ElasticMonitorResource,
+    void,
+    ArmResponse<UpgradableVersionsList>,
+    Error = ResourceProviderDefaultErrorResponse
+  >;
+
+  /**
+   * Upgrade the Elastic monitor resource to a newer version, ensuring optimal observability and performance.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @operationId("Monitor_Upgrade")
+  upgrade is ArmResourceActionAsyncBase<
+    ElasticMonitorResource,
+    ElasticMonitorUpgrade,
+    ArmAcceptedLroResponse,
+    BaseParameters = Azure.ResourceManager.Foundations.DefaultBaseParameters<ElasticMonitorResource>,
+    Error = ResourceProviderDefaultErrorResponse,
+    OptionalRequestBody = true
+  >;
+
+  /**
+   * List all traffic filters associated with your Elastic monitor resource, helping you manage network traffic control.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @action("listAllTrafficFilters")
+  @operationId("AllTrafficFilters_list")
```

Comment:
It should be a separate operation group and should not be included in `ElasticMonitorResources`.